applications/SwimmingDEMApplication/python_scripts/derivative_recovery/standard_recoverer.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
```suggestion
```

can be removed now, just a comment

Reply:
I thought so, but I have not seen it removed in most places. Perhaps this could be done all at once.